core/domain/interaction_jobs_one_off_test.py

```diff
@@ -755,6 +755,224 @@ def test_no_action_is_performed_for_deleted_exploration(self):
+            create_new())
+        interaction_jobs_one_off.MultipleItemInteractionLtOneOffJob.enqueue(
+            job_id)
+        self.process_and_flush_pending_mapreduce_tasks()
+
+        actual_output = (
+            interaction_jobs_one_off
+            .MultipleItemInteractionLtOneOffJob.get_output(job_id))
+        expected_output = (2, 0)
+        self.assertEqual(actual_output, expected_output)
+
+        customization_args_dict2 = {
+            'choices': {'value': [{
+                'html': '<p>This is value1 for MultipleChoiceInput</p>',
+                'content_id': 'ca_choices_0'
+            }, {
+                'html': '<p>This is value2 for MultipleChoiceInput</p>',
+                'content_id': 'ca_choices_1'
+            }, {
+                'html': '<p>This is value3 for MultipleChoiceInput</p>',
+                'content_id': 'ca_choices_2'
+            }, {
+                'html': '<p>This is value4 for MultipleChoiceInput</p>',
+                'content_id': 'ca_choices_3'
+            }]},
+            'showChoicesInShuffledOrder': {'value': True}
+        }
+
+        state2.update_interaction_id('MultipleChoiceInput')
+        state2.update_interaction_customization_args(customization_args_dict2)
+        state2.update_next_content_id_index(4)
+
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        # Start MultipleChoiceInteractionLimitOneOffJob job
+        # on sample exploration whose interaction is
+        # Multiple Choice Input Interaction.
+        job_id = (
+            interaction_jobs_one_off
+            .MultipleItemInteractionLtOneOffJob.
+            create_new())
+        interaction_jobs_one_off.MultipleItemInteractionLtOneOffJob.enqueue(
+            job_id)
+        self.process_and_flush_pending_mapreduce_tasks()
+
+        actual_output = (
+            interaction_jobs_one_off
+            .MultipleItemInteractionLtOneOffJob.get_output(job_id))
+        expected_output = (4, 4)
+        self.assertEqual(actual_output, expected_output)
+
+        state3.update_interaction_id('ItemSelectionInput')
+        state3.update_interaction_customization_args(customization_args_dict1)
+        state3.update_next_content_id_index(6)
+
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        # Start MultipleItemInputInteractionLtOneOffJob job
+        # on sample exploration whose interaction is
+        # Item Selection Input.
+        job_id = (
+            interaction_jobs_one_off
+            .MultipleItemInteractionLtOneOffJob.
+            create_new())
+        interaction_jobs_one_off.MultipleInputInteractionLtOneOffJob.enqueue(
+            job_id)
+        self.process_and_flush_pending_mapreduce_tasks()
+
+        actual_output = (
+            interaction_jobs_one_off
+            .MultipleInputInteractionLtOneOffJob.get_output(job_id))
+        expected_output = (2, 0)
+        self.assertEqual(actual_output, expected_output)
```

Comment:
Can you put this part of code to one common method that will be used by all your tests? Similart to how we do this in other tests for jobs.

Reply:
Done